docs/aurora/programming-models/openmp-aurora.md

```diff
@@ -1 +1,128 @@
 # OpenMP on Aurora
+
+## Overview
+
+The OpenMP API is an open standard for parallel programming. The specification document can be found here: https://www.openmp.org. The specification describes directives, runtime routines, and environment variables that allow an application developer to express parallelism (e.g. shared memory multiprocessing and device offloading). Many compiler vendors provide implementations of the OpenMP specification (https://www.openmp.org/specifications).
+
+## Setting the environment to use OpenMP on Aurora
+
+The Intel oneAPI Programming Environment is the main environment on Aurora to maximally use the hardware. oneAPI has OpenMP support for both CPU threads and GPU devices. The oneAPI module is loaded by default in your environment:
+
+```
+> module list
+
+Currently Loaded Modules:
+  1) gcc/11.2.0                    3) intel_compute_runtime/release/agama-devel-551   5) libfabric/1.15.2.0   7) cray-libpals/1.3.3
+  2) mpich/51.2/icc-all-pmix-gpu   4) *oneapi/eng-compiler/2022.12.30.003*              6) cray-pals/1.3.3
+
+```
+
+However, additional versions of oneAPI with newer compiler versions can be found by adding additional modules to your path:
+
+```
+> module use /soft/modulefiles/
+> module avail oneapi
+
+-------------------------------------------------------------------------------- /soft/modulefiles ---------------------------------------------------------------------------------\
+   oneapi/eng-compiler/2023.05.15.003    oneapi/eng-compiler/2023.10.15.002        oneapi/release/2023.10.15.001        spack-pe-oneapi/0.5-rc1 (D)
+   oneapi/eng-compiler/2023.05.15.006    oneapi/eng-compiler/2023.12.15.002 (D)    oneapi/release/2023.12.15.001 (D)
+   oneapi/eng-compiler/2023.05.15.007    oneapi/release/2023.05.15.001             spack-pe-oneapi/0.4-rc1
+
+------------------------------------------------------------------------- /opt/aurora/23.073.0/modulefiles -------------------------------------------------------------------------\
+   oneapi/eng-compiler/2022.12.30.003 (L)    oneapi/release/2022.12.30.001
+```
+The additional oneAPI modules can be loaded with 'module load oneapi/eng-compiler/2023.10.15.002' for example.
+
+## Building on Aurora
+
+The following table shows the compiler and flags
+
+|language | MPI wrapper compiler (underlying compiler) | flag to turn on OpenMP support and target CPU threads | additional flags to target GPU devices
+| --- | --- | --- | --- |
+| Fortran | mpifort (ifx) | -fiopenmp | -fopenmp-targets=spir64_gen -Xopenmp-target-backend "-device pvc"
```

Comment:
Might want to add code fence backticks into table entries here

Reply:
I made the change and pushed it, thanks!